packages/beat/manifest.yml

```
@@ -0,0 +1,35 @@
+name: beat
+title: Beat
+version: 0.0.1
+description: Beat Integration
+type: integration
+release: experimental
+conditions:
+  kibana:
+    version: ^8.7.0
+owner:
+  github: elastic/infra-monitoring-ui
+format_version: 1.0.0
```

Please use latest package spec version for a new package.
```suggestion
format_version: 2.2.0
```